recipes/sorl-thumbnail/meta.yaml

```
@@ -0,0 +1,47 @@
+{% set name = "sorl-thumbnail" %}
+{% set version = "12.6.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 66771521f3c0ed771e1ce8e1aaf1639ebff18f7f5a40cfd3083da8f0fe6c7c99
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=2.7
+    - pip
+  run:
+    - django >=1.11
+    - python >=2.7
+    - pillow
+
+test:
+  imports:
+    - sorl
+  requires:
+    - django
```

Can we add ``pip check`` here please?
```suggestion
    - pip
  commands:
    - pip check
```